### PR TITLE
cell width fixed

### LINKED ISSUE #13: All tab column width bug
**Describe the bug**
Column width is not equal on all expansion mount tab

**To Reproduce**
1. Go to 'user mounts'
2. Click on 'All' Tab
3. See error

**Expected behavior**
All mount column widths should be equal

**Screenshots**
Column Width

**Desktop (please complete the following information):**
 - OS: Windows

 - Browser Firefox
 - Version 80.0.1

**Smartphone (please complete the following information):**
 - Device: N/A
 - OS: N/A
 - Browser N/A
 - Version N/A

**Additional context**

### LINKED ISSUE #16: Mounts table changes column width as filters or pages change
**Describe the bug**
- When a user changes the expansions checked in the filter or the page number, the column widths change.

**To Reproduce**
1. Go to `Mounts` tab.
2. Click on `A Realm Reborn` checkbox to un-check it.
3. See that the column widths change.
4. Go to the next page in the table.
5. See that the column widths change.

**Expected behavior**
- Column widths should be fixed.

**Screenshots**
gif

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 85.0.4183.83

**Additional context**

### PR DESCRIPTION
**Pull request by EvanM27**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)

## Pull request type

 

Please check the type of change your PR introduces:
- [x] Bug fix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Issue
- [13](https://github.com/sofia819/mount_recorder_client/issues/13)
- [16](https://github.com/sofia819/mount_recorder_client/issues/16)

## What is the current behavior?

- Column widths change when switching between pages.

## What is the new behavior?

- Column widths are fixed when switching between pages and filters.

## Other information

ColumnWidthFixed

### CODE REVIEW

File: src/components/TableCellWrapper.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      display='flex'
+      margin='auto'
+    >
+      {props.content}
+    </Box>
+  );
+};
+
+TableCellWrapper.propTypes = {
+  content: PropTypes.object,
```
Comment (sofia819) [inline src/components/TableCellWrapper.js]: `children: PropTypes.node`

File: src/utils/constants.js [hunk 2]
```diff
@@ -44,6 +44,8 @@ export const LOGIN_BUTTON = 'Login';
 
 export const ADD_USER_BUTTON = 'Add User';
 
+export const COLUMN_WIDTH = '10%';
```
Comment (sofia819) [inline src/utils/constants.js]: Remove

File: src/components/Table.scss [hunk 3]
```diff
@@ -19,3 +19,7 @@
     font-size: 14;
   }
 }
+
+.table-cell {
+  min-width: 100px;
+}
```
Comment (sofia819) [inline src/components/Table.scss]: Remove

File: src/components/TableCellWrapper.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Box } from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+export const TableCellWrapper = (props) => {
```
Comment (sofia819) [inline src/components/TableCellWrapper.js:6]: - Add a type prop to decide what heights and widths the Box should be.
- Declare a constant to map types to heights and widths.

File: src/components/UserMounts/UserMountsTable.js [hunk 5]
```diff
@@ -7,23 +7,24 @@ import {
...
+import { TableCellWrapper } from 'components/TableCellWrapper';
+import PropTypes from 'prop-types';
+import 'components/Table.scss';
 
 export const UserMountsTable = (props) => {
   return (
     <TableContainer component={Paper}>
-      <Table className="table" aria-label="customized table">
-        <TableHead className="table-head">
+      <Table className='table' aria-label='customized table'>
```
Comment (sofia819) [inline src/components/UserMounts/UserMountsTable.js]: Go over the styles and remove classes that don't exist anymore

**Review by sofia819 — COMMENTED** [review]
Some ideas

File: src/components/UserMounts/UserMountsTable.js (hunk 5 above)
Comment (EvanM27) [reply src/components/UserMounts/UserMountsTable.js]: extraneous styles and classes eliminated

File: src/components/Table.scss (hunk 3 above)
Comment (EvanM27) [reply src/components/Table.scss]: Removed

File: src/utils/constants.js (hunk 2 above)
Comment (EvanM27) [reply src/utils/constants.js]: Removed

File: src/components/TableCellWrapper.js (hunk 1 above)
Comment (EvanM27) [reply src/components/TableCellWrapper.js]: PropType added

File: src/components/TableCellWrapper.js (hunk 4 above)
Comment (EvanM27) [reply src/components/TableCellWrapper.js:6]: Basic design implemented

File: src/components/TableCellWrapper.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+      display='flex'
+      margin='auto'
+    >
+      {props.children}
+    </Box>
+  );
+};
+
+TableCellWrapper.propTypes = {
+  children: PropTypes.node,
```
Comment (sofia819) [inline src/components/TableCellWrapper.js:23]: PropType for `type`?

File: src/index.js [hunk 7]
```diff
@@ -1,3 +1,6 @@
+//////////////////////////////////////////////
+///  Copyright Web Static initiative 2020  ///
+//////////////////////////////////////////////
```
Comment (sofia819) [inline src/index.js]: lol

File: src/utils/constants.js [hunk 8]
```diff
@@ -46,6 +48,27 @@ export const ADD_USER_BUTTON = 'Add User';
 
 export const EXPANSION_COLUMN = 'Expansion';
 
+export const USER_CELL_DIMENSIONS = 0;
+
+export const MOUNT_CELL_DIMENSIONS = 1;
+
+export const EXPANSION_CELL_DIMENSIONS = 2;
```
Comment (sofia819) [inline src/utils/constants.js]: Remove?

**Review by sofia819 — APPROVED** [review]
Minor comments but LGTM

File: src/utils/constants.js (hunk 8 above)
Comment (EvanM27) [reply src/utils/constants.js]: removed

File: src/components/TableCellWrapper.js (hunk 6 above)
Comment (EvanM27) [reply src/components/TableCellWrapper.js:23]: included `type: PropTypes.string,`

File: src/index.js (hunk 7 above)
Comment (EvanM27) [reply src/index.js]: It's good stuff

**Pull request merged**